mautic/mautic
# Segment - Fix filter URL Visited with regex and more

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [Y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [Y]
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | Fixes #11427 


#### Description:

This PR fixes the problem with regex filters, like the visited url.
The filter did not take into account the regex, and display the same number of contacts no matter the regex.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Login
4. Create two segments with filter "Visited URL" and "regexp", but not same expression (depends on your current data in table page_hits)
5. Update your two segments with command
6. You will see that the filters work
7. Enjoy



<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11541"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

PR funded by @PierreAmmeloot from [WebAnyOne](https://www.webanyone.net/)
